src/core/datatable.js

```diff
@@ -178,6 +178,12 @@ export const getColumnsFilterOptions = ({
           }
         }
       });
+      if (_columnsFilterOptions[columnIndex]) {
+        _columnsFilterOptions[columnIndex].sort();
```

Comment:
```suggestion
```
Why are we sorting twice? I think this line can be removed and it will still work fine.